apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -125,13 +149,12 @@ function HomeScreenInner({ account }: { account: Account }) {
           <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
         }
         contentContainerStyle={{
-          height:
-            screenDimensions.height -
-            tabBarHeight -
-            (Platform.OS === "android" ? 16 : 0),
+          height: screenHeight,
         }}
         onScrollBeginDrag={onScrollBeginDrag}
         onScrollEndDrag={onScrollEndDrag}
+        onScroll={scrollHandler}
+        scrollEventThrottle={8}
```

Comment:
any reason to make this lower than 16?

Reply:
Talked Reanimated maintainer asking what will happen on 120hz screens if it only happened every 16ms, and the answer was: choppy experience

In the new release of reanimated it gonna adapt to the screen refresh rate but for 3.3.0 the best idea is to use 8 for now

Reply:
Sounds good!